subprojects/docs/src/main/resources/header.html

```diff
@@ -181,15 +181,22 @@ <h3 id="authoring-gradle-builds">Authoring Gradle Builds</h3>
                     <li><a href="../userguide/potential_traps.html">Avoiding Traps</a></li>
                 </ul>
             </li>
-            <li><a class="nav-dropdown" data-toggle="collapse" href="#authoring-multi-project-builds" aria-expanded="false" aria-controls="authoring-multi-project-builds">Authoring Multi-Project Builds</a>
-                <ul id="authoring-multi-project-builds">
+            <li><a class="nav-dropdown" data-toggle="collapse" href="#authoring-multi-project-builds" aria-expanded="false" aria-controls="authoring-multi-project-builds">Structuring a S/W Component</a>
+                <ul id="structuring-software-component">
                     <li><a href="../userguide/multi_project_builds.html">Creating a Basic Multi-Project Build</a></li>
                     <li><a href="../userguide/declaring_dependencies_between_subprojects.html">Declaring Dependencies between Subprojects</a></li>
                     <li><a href="../userguide/sharing_build_logic_between_subprojects.html">Sharing Build Logic between Subprojects</a></li>
                     <li><a href="../userguide/fine_tuning_project_layout.html">Fine Tuning the Project Layout</a></li>
                     <li><a href="../userguide/multi_project_configuration_and_execution.html">Understanding Configuration and Execution</a></li>
                 </ul>
             </li>
+            <li><a class="nav-dropdown" data-toggle="collapse" href="#authoring-multi-project-builds" aria-expanded="false" aria-controls="authoring-multi-project-builds">Structuring a S/W Product</a>
```

Comment:
THis looks like it has the same href as the one above?

Reply:
👍 Good catch!